### PR TITLE
WSTEAM1-779 Live Page Promo Image

### PR DESCRIPTION
**Pull request by emilysaffron**
Resolves JIRA [WSTEAM1-779](https://jira.dev.bbc.co.uk/browse/WSTEAM1-779)

Overall changes
======
Adds promo image to SEO schema when it is present

Code changes
======

- Extracts `promoImage` from`pageData` and returns its `url` in the `LinkedDataContainer`

Testing
======
1. **Live page with promo image** - Visit http://localhost:7081/urdu/live/cd0jyl9r0lzt?renderer_env=test and observe the thumbnail url within the SEO schema (using detailed SEO extension or by checking the meta tags of the page) matches the one within [the FABL response](https://fabl.api.bbci.co.uk/playground?env=production&module=simorgh-bff&context=ctx-service-env%3Dtest&params=id%3Dcd0jyl9r0lzt%3Btrue%2Cservice%3Durdu%3Btrue%2CpageType%3Dlive%3Btrue)
2. **Live page without promo image** - Visit http://localhost:7081/pidgin/live/c7p765ynk9qt?renderer_env=test and observe that the thumnail url is the default service logo, as promo image is null in[ the FABL response](https://fabl.api.bbci.co.uk/playground?env=production&module=simorgh-bff&context=ctx-service-env%3Dtest&params=id%3Dc7p765ynk9qt%3Btrue%2Cservice%3Dpidgin%3Btrue%2CpageType%3Dlive%3Btrue)
3. **Non live page**- Visit http://localhost:7080/hausa/articles/c2nr6xqmnewo and observe that the thumnail url is the default service logo , as before - unaffected by this change.

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

**Comment by amoore108** [comment]
May be worth checking with Tom if this should be setting the `<meta />` Opengraph tags (which is done via the MetadataContainer component) instead of the Schema.org JSON? I thought thats what it was meant to do for social sharing, but I could be wrong.

**Comment by emilysaffron** [comment]
> May be worth checking with Tom if this should be setting the `<meta />` Opengraph tags (which is done via the MetadataContainer component) instead of the Schema.org JSON? I thought thats what it was meant to do for social sharing, but I could be wrong.

ohhh yeah that makes sense, will double check !

**Comment by emilysaffron** [comment]
> > May be worth checking with Tom if this should be setting the `<meta />` Opengraph tags (which is done via the MetadataContainer component) instead of the Schema.org JSON? I thought thats what it was meant to do for social sharing, but I could be wrong.
> 
> ohhh yeah that makes sense, will double check !

Checked with tom , he clarified that the Schema will add the promo image to google and other search engines like this:
Screenshot 2024-03-11 at 16 27 53

And from what he said and info [here](https://kaydee.net/blog/open-graph-image/)  the open graph tags add the preview images in facebook and twitter , so think we need both :)

File: ws-nextjs-app/pages/[service]/live/[id]/LivePageLayout.tsx [hunk 1]
```diff
@@ -108,6 +111,7 @@ const LivePage = ({ pageData }: ComponentProps) => {
       <MetadataContainer
         title={pageTitle}
         lang={lang}
+        promoImage={promoImage}
```
Comment (amoore108) [inline ws-nextjs-app/pages/[service]/live/[id]/LivePageLayout.tsx]: Can we use the existing `image`, `imageAltText`, `imageWidth` and `imageHeight` props to pass in the promo image? Would save us having to do the checks and conditional renders within the `Metadata` component.

Comment (emilysaffron) [reply ws-nextjs-app/pages/[service]/live/[id]/LivePageLayout.tsx]: yes good idea!

**Review by amoore108 — APPROVED** [review]

**Review by Isabella-Mitchell — APPROVED** [review]

**Review by Isabella-Mitchell — COMMENTED** [review]
Wondering if we need to add/ update any unit tests?

**Comment by emilysaffron** [comment]
> Wondering if we need to add/ update any unit tests?

Guess if we were gonna they would need to be added [here](https://github.com/bbc/simorgh/blob/latest/src/app/components/LinkedData/index.test.tsx) as nothing else was really changed functionality wise, but looking in there there isn't a precedent for checking if the thumnail url is the correct one .. so think we're okay ?

**Pull request merged**